### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by mphfish**

### CODE REVIEW

**Review by toland — APPROVED** [review]
LGTM

File: mix.exs [hunk 1]
```diff
@@ -42,7 +42,6 @@ defmodule Dawdle.MixProject do
       env: [
         backend: {:system, :module, "DAWDLE_BACKEND", Dawdle.Backend.Local},
         start_pollers: {:system, :boolean, "DAWDLE_START_POLLERS", false},
-        forward_raw_messages: {:system, :boolean, "DAWDLE_FORWARD_RAW_MESSAGES", false},
```
Comment (dustinfarris) [inline mix.exs:45]: why is this being removed?

Comment (mphfish) [reply mix.exs:45]: I have no idea what you're talking about 👀

**Review by dustinfarris — APPROVED** [review]

File: mix.exs (hunk 1 above)
Comment (mphfish) [reply mix.exs:45]: I migrated these changes from my own fork and accidentally included this.

**Pull request merged**